plugin/install.go

```diff
@@ -105,12 +110,44 @@ func doPluginInstall(c *cli.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "Failed to install plugin while making a download URL")
 	}
+
+	canUpgrade := true
```

Comment:
This variable means that meta data store is enabled or not.  So I think using `isMetaDataStoreEnabled` as variable name is better.  How is it?

Reply:
fixed in fc3b81d3190e60262635d9a787670c6e0eb8a84b